.github/workflows/quality.yml

```diff
@@ -23,4 +23,23 @@ jobs:
           pip install pre-commit
       - name: Run pre-commit
         run: pre-commit run --all-files
-  
\ No newline at end of file
+
+
+  mypy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.12"
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install dependencies
```

Comment:
added installing dependencies and then running mypy for this workflow.

should this be a pre-commit instead?

Reply:
No, I wouldn't install dependencies via pre-commit.

Furthermore, I suggest to create testing, linting and other dependency environments so that you can install them with scoped versions from those files instead of adding them in the CI like here.

We can have a look at that in our call :)

Reply:
Awesome.

Reply:
referenced this in new [Issue 34](https://github.com/Tanner-Ray-Martin/Dynamic-DB/issues/34)